base/applications/mspaint/dialogs.cpp

```diff
@@ -198,7 +204,7 @@ LRESULT CAttributesDialog::OnEdit2(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOO
         else if (IsDlgButtonChecked(IDD_ATTRIBUTESRB2))
```

Comment:
Side-question: What's the difference between those two OnEdit1 and OnEdit2 functions? It seems their code is identical -- except for the control ID of one single edit control.
(It seems possible one of those functions can be removed, and take an extra parameter to specify the edit control ID to act upon.)

Reply:
There has one more difference: `newWidth` and `newHeight`.
I think no need to commonize this functions.